src/sql/src/vars.rs

```diff
@@ -412,6 +412,18 @@ const PERSIST_COMPACTION_MINIMUM_TIMEOUT: ServerVar<Duration> = ServerVar {
     safe: true,
 };
 
+/// Controls whether or not to use the new storage `persist_sink` implementation in storage
+/// ingestions.
+const ENABLE_MULTI_WORKER_STORAGE_PERSIST_SINK: ServerVar<bool> = ServerVar {
+    name: UncasedStr::new("enable_multi_worker_storage_persist_sink"),
+    value: &false,
+    description: "Whether or not to use the new multi-worker storage `persist_sink` \
+                  implementation in storage ingestions. Is applied only \
+                  when a cluster or dataflow is restarted.",
```

Comment:
We usually end the description of Materialize vars with `(Materialize).`, see above.

Reply:
Oh this is here because its stacked on top of this pr: https://github.com/MaterializeInc/materialize/pull/18010, so ill fix it there!